detect_secrets/plugins/private_key.py

```diff
@@ -39,16 +40,31 @@ class PrivateKeyDetector(RegexBasedDetector):
 
     secret_type = 'Private Key'
 
+    begin_key_opening = r'(?P<begin_key>BEGIN'
+    key_types = r'(?: DSA | EC | OPENSSH | PGP | RSA | SSH2 ENCRYPTED | )'
+    begin_key_closing = r'PRIVATE KEY-*)'
+    begin_key = begin_key_opening + key_types + begin_key_closing
```

Comment:
```suggestion
    begin_key = fr"{begin_key_opening}{key_types}{begin_key_closing}"
```
this will create only one raw string 🙂

Reply:
 Cool :)